src/routes/products/[barcode]/ProductHeader.svelte

```
@@ -252,6 +256,17 @@
 				{/if}
 
 				<!-- Traceability Codes -->
+				<!-- Manufacturing Places -->
+				{#if product.manufacturing_places}
+					<div class="mb-2">
+						<div class="text-secondary mb-2 text-sm font-bold">Manufacturing Places</div>
+						<div class="flex flex-wrap justify-center gap-1 md:justify-start">
+							{#each product.manufacturing_places.split(',').filter((p) => p.trim()) as place}
+								<span class="badge wrap-break-word">{place.trim()}</span>
+							{/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)